event_sync/utils.go

```diff
@@ -0,0 +1,48 @@
+package event_sync
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+)
+
+func callAPIPost(path string, req, resp interface{}) error {
```

Comment:
nit: I might shorten to just `postAPI` (and `getAPI` below), but I don't feel strongly.